### DISCUSSION TITLE
My Copilot Experience

### DISCUSSION THREAD

**Discussion by jabo-bernardo** [community/community#7425] [Copilot Conversations]
I've been using Github Copilot for quite a few days now and I must say that it really helped speed up my development & boost my productivity I haven't made a single StackOverflow or google search since I used it. At first, I looked at this as a threat but after a few days, I considered it as more of a tool than a threat. I'm looking forward to seeing where this tool go in the next few years.

**Comment by SmallLemur** [comment]
@jabo-bernardo thanks for the  feedback ! Would you mind telling me if you are still using Copilot ?and if so, I would be very interested to understand better your use case.

**Reply by jabo-bernardo** [reply]
Hey, yes I'm still using Copilot and I'm still enjoying it. I mainly use it for web development and it works perfectly with eac.Hey, yes I'm still using Copilot and so far it is amazing. I mainly use it for web development it works perfectly with the common front-end javascript frameworks and back-end node.js development.

**Reply by SmallLemur** [reply]
Awesome, would you be interested to have a quick chat ? I would like to hear more about your Copilot experience :)
